apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -1809,14 +1809,14 @@ var (
 			CoreDNS:                   m("coredns/coredns:1.3.1"),
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.3.0"),
 		},
-		// Enabled in Rancher v2.2.11
-		"v1.15.11-rancher1-0": {
+		// Enabled in Rancher v2.2.13
+		"v1.15.12-rancher1-1": {
 			Etcd:                      m("quay.io/coreos/etcd:v3.3.10-rancher1"),
-			Kubernetes:                m("rancher/hyperkube:v1.15.11-rancher1"),
-			Alpine:                    m("rancher/rke-tools:v0.1.50"),
-			NginxProxy:                m("rancher/rke-tools:v0.1.50"),
-			CertDownloader:            m("rancher/rke-tools:v0.1.50"),
-			KubernetesServicesSidecar: m("rancher/rke-tools:v0.1.50"),
+			Kubernetes:                m("rancher/hyperkube:v1.15.12-rancher1"),
+			Alpine:                    m("rancher/rke-tools:v0.1.56"),
```

Comment:
Why are RKE tools changing?

Reply:
This is new k8s version, so has the latest RKE tools introduced with it, https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/k8s_rke_system_images.go#L1986